src/Domain/BlockId.hpp

```diff
@@ -0,0 +1,68 @@
+// Distributed under the MIT License.
+// See LICENSE.txt for details.
+
+#pragma once
+
+#include <cstddef>
+#include <ostream>
+#include <pup.h>
+
+namespace PUP {
+class er;
+}  // namespace PUP
+
+namespace domain {
```

Comment:
Depends on #814.

Reply:
How is this possibly dependent on that? It just adds a new class...